framework/src/main/java/org/checkerframework/common/initializedfields/qual/InitializedFieldsPredicate.java

```diff
@@ -0,0 +1,25 @@
+package org.checkerframework.common.initializedfields.qual;
+
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import java.lang.annotation.Target;
+import org.checkerframework.framework.qual.SubtypeOf;
+
+/**
+ * A predicate annotation for the Initialized Fields type system.
+ *
+ * @checker_framework.manual #initialized-fields-checker Initialized Fields Checker
+ */
+@Retention(RetentionPolicy.RUNTIME)
+@Target({ElementType.TYPE_USE, ElementType.TYPE_PARAMETER})
+@SubtypeOf({InitializedFields.class})
+public @interface InitializedFieldsPredicate {
```

Comment:
Why do you need `InitializedFieldsPredicate`? I can't think of a situation where it would be useful.

Reply:
(There is a constructor in `AccumulationAnnotatedTypeFactory` that doesn't require a predicate annotation.)